test/schema.sql

```diff
@@ -276,6 +276,7 @@ CREATE TABLE "user" (
     email character varying(100)
 );
 
+ALTER TABLE "user" ADD CONSTRAINT user_email_unique UNIQUE (email);
```

Comment:
Is it better to:

```
ALTER TABLE "user" ADD UNIQUE (email);
```

Reference: https://www.beekeeperstudio.io/blog/guide-to-unique-constraints-in-postgresql

Reply:
Hi, @dhmlau. I have changed the query. Please have a look now,